src/app/app.component.html

```diff
@@ -32,6 +32,11 @@
             routerLink="/ots"
             routerLinkActive="active" (click)="clicked($event)">About This Site</a>
       </li>
+      <li *ngIf="enableMl" class="nav-item">
+        <a class="nav-link nav-link-custom"
+            routerLink="/ml"
+            routerLinkActive="active" (click)="clicked($event)">ML</a>
+      </li>
     </ul>
     <!-- <form class="form-inline my-2 my-lg-0">
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)